jina/executors/encoders/nlp/transformer.py

```diff
@@ -94,6 +94,9 @@ def post_init(self):
         self._sess_func = None
         self.tmp_model_path = self.model_abspath if os.path.exists(self.model_abspath) else self.model_name
         self._tokenizer = self.get_tokenizer()
+
+        self._has_cls_token = True if getattr(self._tokenizer, 'cls_token', None) is not None else False
```

Comment:
very redundant logic

```suggestion
        self._has_cls_token = hasattr(self._tokenizer, 'cls_token')
```